Print_A_Bot/Print_A_Bot/settings.py

```diff
@@ -38,9 +39,12 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
 
+    'djconfig',
```

Comment:
used by previous project, doing it over [non required task](http://taiga.mitescreations.com/project/alvin-print-a-bot/us/19?kanban-status=43) I'd use a static json file